### DISCUSSION TITLE
Delta file of only modified changes not new changes.

### DISCUSSION THREAD

**Discussion by robot-cpu** [scolladon/sfdx-git-delta#609] [Q&A]
Is there anyway to create package of modified files but not include new changes in that delta file?

**Comment by scolladon** [comment]
Hi @robot-cpu 

There is no way to do that.
We do not have implemented filtering based on git type of changes.
You can filter based on file modified.

What is your use case here ? In which situation it could become handy to filter on a git change type ?
(it could be a great feature)

**Reply by robot-cpu** [reply]
your plugin did the work but still need to make some changes for my own file structure. is there any documentation to understand more about main.js is working with json and other files

**Reply by scolladon** [reply]
No particular documentation for those file except the unit tests.
If you'd like we could schedule a call so I can try answer your questions, see if I can help you with your challenges and see if we could add new feature to help you solve them.
